jobs/build-podman-os.Jenkinsfile

```
@@ -0,0 +1,355 @@
+                pipeutils.withOptionalExistingCosaRemoteSession(
+                arch: arch, session: archinfo[arch]['session']) {
+                    parallel artifacts.collectEntries{artifact -> [artifact["platform"] , {
+                        def file = "podman-$arch-${shortcommit}.${artifact["suffix"]}"
+                        shwrap("""cosa shell -- /usr/bin/zstd -14 $file""")
+                    }]}
+                }
+            }]}
+        }
+        stage("Sync Artifacts") {
+            parallel archinfo.keySet().collectEntries{arch -> [arch, {
+                   pipeutils.withOptionalExistingCosaRemoteSession(
+                   arch: arch, session: archinfo[arch]['session']) {
+                       if (arch != "x86_64") {
+                           parallel artifacts.collectEntries{artifact -> [artifact["platform"] , {
+                               shwrap("""cosa remote-session sync {:,}podman-$arch-${shortcommit}.${artifact["suffix"]}.zst """)
+                           }]}
+                       }
+                   }
+            }]}
+        }
+        withCredentials([file(credentialsId: 'ravanelli-push-registry-secret', variable: 'REGISTRY_SECRET')]) {
+            stage("Push Manifest") {
+                def manifest = "${params.CONTAINER_REGISTRY_STAGING_REPO}:5.0-${shortcommit}"
+                def cmds = ""
+                parallel archinfo.keySet().collectEntries{arch -> [arch, {
+                    def image = "docker://${params.CONTAINER_REGISTRY_STAGING_REPO}:${arch}-${shortcommit}"
+                    cmds += " && /usr/bin/buildah manifest add ${manifest} ${image}"
+                    artifacts.each { artifact ->
+                        def file = "podman-$arch-${shortcommit}.${artifact["suffix"]}.zst"
+                        def platform = "${artifact["platform"]}"
+                        cmds += " && /usr/bin/buildah manifest add --artifact --artifact-type='' \
+                                --os=linux --arch=${arch} --annotation 'disktype=${platform}' ${manifest} ${file}"
+                    }
+                }]}
+                //  Supermin is unable to access the secret mounted in another directory.
+                // Although this solution is not ideal, let's proceed with it until we
+                // can find a more effective approach.
+                // This doc includes a 'use' case that didn't function as expected.
+                // https://www.jenkins.io/doc/pipeline/steps/credentials-binding/
+                shwrap("""cp ${REGISTRY_SECRET} push-secret""")
+
+                // We need to run all buildah commands in the same superminvm to not lose
+                // the manifest creation/additions
+                shwrap("""cosa supermin-run bash -c \
+                    "/usr/bin/buildah manifest create ${manifest} ${cmds} && \
+                    /usr/bin/buildah manifest push --authfile ./push-secret --all ${manifest} \
+                    docker://${manifest}"
```

is the `docker://` required here? 
```suggestion
                    ${manifest}"
```

Looking at the COSA code that pushes I don't think we have `docker://` on there.

---

If I recall buildah complains about the transport if we don't add it.

---

ok